app/components/Search.js

```diff
@@ -120,24 +121,78 @@ class Search extends Component {
 }
 
 const styles = StyleSheet.create({
+  container: {
+    flex: 1,
```

Comment:
We probably do styling differently, but I like to arrange mine in alphabetical order.